CrowServe/CrowServe.cpp

```diff
@@ -78,8 +78,17 @@ bool Socket::receive() {
             std::cout << "received message header, type: " << message_header.message_type << " size: " << message_header.message_size << "\n";
             // TODO: figure out protocol-specific message handling
             char buffer[1600];
+            auto p = &buffer;
             auto received = recv(m_socket, &buffer, message_header.message_size, 0);
-            std::cout << "received message of size " << received << "\n";
+            auto sp = std::span<const char>{buffer, (u32)received};
```

Comment:
Not important, but let's not get carried away with one letter vars

Reply:
This function is replaced by `listen` so it's only being used for prototyping now. Basically until I have the integration test set up I'm using it to try things out.